docker-compose.yml

```diff
@@ -62,6 +62,9 @@ services:
       - "./php:/var/www/memoires-info/php/"
     depends_on:
       - postg-rest
+    env_file:
```

Comment:
Good idea, this will work. But we do not want to pass all env to all containers, this can lead to security concerns. In our case we want to pass only the required environment variables. https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/#use-the-environment-attribute